src/ipahealthcheck/core/core.py

```diff
@@ -157,6 +157,9 @@ def add_default_options(parser, output_registry, default_output):
                         default=default_output, help='Output method')
     parser.add_argument('--output-file', dest='outfile', default=None,
                         help='File to store output')
+    parser.add_argument('--version', dest='version', action='store_true',
+                        default=False,
```

Comment:
`default` is already False for `action='store_true`:
https://docs.python.org/3/library/argparse.html#action

Reply:
`dest='version'` is also redundant.

Reply:
Yeah I tend to just keep things explicit. I can remove it.

Reply:
Yes, please, remove.